Doc/library/ssl.rst

```diff
@@ -1952,19 +1954,20 @@ If you prefer to tune security settings yourself, you might create
 a context from scratch (but beware that you might not get the settings
 right)::
 
-   >>> context = ssl.SSLContext()
-   >>> context.verify_mode = ssl.CERT_REQUIRED
-   >>> context.check_hostname = True
+   >>> context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
    >>> context.load_verify_locations("/etc/ssl/certs/ca-bundle.crt")
 
 (this snippet assumes your operating system places a bundle of all CA
 certificates in ``/etc/ssl/certs/ca-bundle.crt``; if not, you'll get an
 error and have to adjust the location)
 
 When you use the context to connect to a server, :const:`CERT_REQUIRED`
-validates the server certificate: it ensures that the server certificate
-was signed with one of the CA certificates, and checks the signature for
-correctness::
+validates the server certificate and :meth:`~SSLContext.check_hostname`
+matches the hostname. Both setting ensure that the server certificate
+was signed with one of the CA certificates and is a valid certificate
+for the given server name. The :data:`PROTOCOL_TLS_CLIENT` protocol
+configures the context for cert and hostname verification. All
```

Comment:
This sentences seems like it should be the first on in the paragraph, since it's explaining the previous lines of code.